src/App.test.tsx

```diff
@@ -16,3 +21,83 @@ describe("Welcome Page", () => {
     ReactDOM.render(<WelcomePage start={undefined} />, div);
   });
 });
+
+describe("Select Quiz", () => {
+  it("renders without crashing", () => {
+    const div = document.createElement("div");
+    ReactDOM.render(
+      <SelectQuiz
+        startQuiz={undefined}
+        selectQuizArr={[10, 25, 50, 100, 681]}
+      />,
+      div
+    );
+  });
+  it("Has a button for every quiz question amount under 600", () => {
+    const { getByText } = render(
+      <SelectQuiz
+        startQuiz={undefined}
+        selectQuizArr={[10, 25, 50, 100, 599]}
+      />
+    );
+    expect(getByText("10").textContent).toBeDefined();
+    expect(getByText("25").textContent).toBeDefined();
+    expect(getByText("50").textContent).toBeDefined();
+    expect(getByText("100").textContent).toBeDefined();
+    // TODO: Ensure it isn't supposed to be All (599) here
+    expect(getByText("599").textContent).toBeDefined();
+  });
+  it("Has a button for every quiz question amount for max amount equal to 600", () => {
+    const { getByText } = render(
+      <SelectQuiz
+        startQuiz={undefined}
+        selectQuizArr={[10, 25, 50, 100, 600]}
+      />
+    );
+    expect(getByText("10").textContent).toBeDefined();
+    expect(getByText("25").textContent).toBeDefined();
+    expect(getByText("50").textContent).toBeDefined();
+    expect(getByText("100").textContent).toBeDefined();
+    // TODO: Ensure it isn't supposed to be All (600) here
+    expect(getByText("600").textContent).toBeDefined();
+  });
+  it("Has a button for every quiz question for max amount equal to 601", () => {
+    const { getByText } = render(
+      <SelectQuiz
+        startQuiz={undefined}
+        selectQuizArr={[10, 25, 50, 100, 601]}
+      />
+    );
+    expect(getByText("10").textContent).toBeDefined();
+    expect(getByText("25").textContent).toBeDefined();
+    expect(getByText("50").textContent).toBeDefined();
+    expect(getByText("100").textContent).toBeDefined();
+    expect(getByText("All (601)").textContent).toBeDefined();
+  });
+});
```

Comment:
I think we should simplify these tests for the Select Quiz because seems repetitive right now. 

Maybe something along these lines to make it cleaner and less repetitive. 
```

  it("Has a button for every quiz question amount under 600", () => {
    const { getByText } = render(
      <SelectQuiz startQuiz={undefined} selectQuizArr={[10, 25, 50, 100]} />
    );
    expect(getByText("10").textContent).toBeDefined();
    expect(getByText("25").textContent).toBeDefined();
    expect(getByText("50").textContent).toBeDefined();
    expect(getByText("100").textContent).toBeDefined();
  });

  it("Has a button for max amount equal to 600 and 601", () => {
    const { getByText } = render(
      <SelectQuiz startQuiz={undefined} selectQuizArr={[600, 601]} />
    );

    expect(getByText("600").textContent).toBeDefined();
    expect(getByText("All (601)").textContent).toBeDefined();
  });
```

Reply:
Or you can simplify that last test to just check for 601

Reply:
I decided to rewrite the Select Quiz render method to not use 601 as the arbitrary cut-off point. It appears everything is still working properly however.